app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -292,6 +296,31 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false
     )
 
+    /**
+     * Shows if the user has chosen to close the set default browser experiment card
+     * on home screen or has clicked the set as default browser button.
+     */
+    var userDismissedExperimentCard by booleanPreference(
+        appContext.getPreferenceKey(R.string.pref_key_experiment_card_home),
+        default = false
+    )
+
+    /**
+     * Shows if the set default browser experiment card should be shown on home screen.
+     */
+    fun shouldShowSetAsDefaultBrowserCard(): Boolean {
+        val browsers = BrowsersCache.all(appContext)
+        val experiments = appContext.components.analytics.experiments
+        val isExperimentBranch =
+            experiments.withExperiment(Experiments.DEFAULT_BROWSER) { experimentBranch ->
+                (experimentBranch == ExperimentBranch.DEFAULT_BROWSER_NEW_TAB_BANNER)
+            }
+        return isExperimentBranch &&
+                !userDismissedExperimentCard &&
```

Comment:
I wonder if the experiment should only be shown once? As after the experiment has been dismissed one time, we are not going to allow it anymore.

Reply:
According to https://github.com/mozilla-mobile/fenix/issues/18375#issuecomment-815254448 the card should be shown every time after the 3rd app launch until the user dismisses the card.

Reply:
Thanks for the clarification 👍🏽